yt_dlp/extractor/bilibili.py

```diff
@@ -164,14 +164,18 @@ def _sign_wbi(self, params, video_id):
         params['w_rid'] = hashlib.md5(f'{query}{self._get_wbi_key(video_id)}'.encode()).hexdigest()
         return params
 
-    def _download_playinfo(self, bvid, cid, headers=None, qn=None):
-        params = {'bvid': bvid, 'cid': cid, 'fnval': 4048}
-        if qn:
-            params['qn'] = qn
+    def _download_playinfo(self, bvid, cid, headers=None, query={}):
+        params = {'bvid': bvid, 'cid': cid, 'fnval': 4048, **query}
```

Comment:
a nitpick since we unpack it immediately and don't use it again, but it's better to not have mutable args IMO
```suggestion
    def _download_playinfo(self, bvid, cid, headers=None, query=None):
        params = {'bvid': bvid, 'cid': cid, 'fnval': 4048, **(query or {})}
```